modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectDefinitionUtil.java

```diff
@@ -91,7 +98,9 @@ public static ObjectDefinition toObjectDefinition(
 			serviceBuilderObjectDefinition,
 		SystemObjectDefinitionManagerRegistry
 			systemObjectDefinitionManagerRegistry,
-		UserLocalService userLocalService,
+		long userId, UserLocalService userLocalService,
+		PermissionService permissionService,
+		ResourceActionLocalService resourceActionLocalService,
```

Comment:
should be sorted:
Portal portal, PermissionService permissionService,
ResourceActionLocalService resourceActionLocalService,
...
SystemObjectDefinitionManagerRegistry systemObjectDefinitionManagerRegistry,				
long userId, UserLocalService userLocalService